com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/model/SupportedParameters.java

```diff
@@ -34,6 +34,7 @@
     public static final String GENERATED_PASSWORD = "generated-password";
     public static final String DEFAULT_IDLE_DOMAIN = "default-idle-domain";
     public static final String TIMESTAMP = "timestamp";
+    public static final String PARALLEL_DEPLOYMENTS = "parallel-deployments";
```

Comment:
Rename this to "enable-parallel-deployments". This would be consistent with the "enable-ssh" parameter. Actually, after reviewing the ComputeNextAppsStep - rename this to "enable-parallel-operations", since "operations" groups "deployments" and "restarts".